teps/0089-nonfalsifiable-provenance-support.md

```diff
@@ -262,10 +264,10 @@ We'll also need this feature for Tekton to achieve [SLSA 3](https://slsa.dev/lev
 As mentioned above, the basic design looks like this (this is meant to be high level and still needs to be fleshed out a bit):
 1. Tekton Pipelines receives a TaskRun config, and generates the Pod for it with SPIRE mounted in
 1. The Pod executes, and the entrypointer requests an SVID and signature over the Results
-1. Tekton Pipelines verifies the Results against the SPIRE SVID and Trust Bundle
+1. Tekton Pipelines verifies the Results against the SPIRE SVID and Trust Bundle and sets the `TaskRunResultsVerified` condition to `True`.
```

Comment:
nit: i think this is called `SignedResultsVerified` in the examples below

Reply:
Yup - i will fix this! Good catch - we've been bouncing around the 2 too much!